yarn.lock

```diff
@@ -3331,6 +3475,18 @@
     "@types/source-list-map" "*"
     source-map "^0.6.1"
 
+"@types/webpack@^4":
```

Comment:
Not particularly happy that it pulls in the old webpack 4 types.. but this should go away as modules get updated